packages/@aws-cdk/aws-servicecatalogappregistry/lib/application-associator.ts

```diff
@@ -28,6 +28,14 @@ export interface ApplicationAssociatorProps {
    */
   readonly description?: string;
 
+  /**
+   * Stack ID.
+   *
+   * @default - ApplicationAssociatorStack
+   *
+   */
+  readonly stackId?: string;
```

Comment:
```suggestion
  readonly applicationStackId?: string;
```
I think that just stackId here may be confusing. Let's make the field name more specific.

Reply:
Shouldn't we also rename `stackProps` to `applicationStackProps` then?

Reply:
I think so? Given your role, I think you would have a better idea of the user experience here so I'm wondering what your thoughts are on this change. I prefer it for clarity but I don't use this service.